test/system/kind.spec.ts

```diff
@@ -139,19 +123,15 @@ test('Kubernetes-Monitor with KinD', async (jestDoneCallback) => {
     .reply(
       200,
       (uri, requestBody: transmitterTypes.IClusterMetadataPayload) => {
-        try {
-          expect(requestBody).toEqual<
-            Partial<transmitterTypes.IClusterMetadataPayload>
-          >({
-            agentId,
-            cluster: expect.any(String),
-            userLocator: expect.any(String),
-            // also should have version here but due to test limitation it is undefined
-            // as it is injected as an environment variable via the Helm chart
-          });
-        } catch (error) {
-          jestDoneCallback(error);
-        }
+        expect(requestBody).toEqual<
+          Partial<transmitterTypes.IClusterMetadataPayload>
+        >({
+          agentId: expect.any(String),
```

Comment:
why are we not checking the agentId matches the one given anymore?

Reply:
I should revert that probably. Good catch